src/app/cube/details/components/outcome/outcome.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit, Input, ChangeDetectorRef } from '@angular/core';
 import { LearningOutcome } from '@entity';
 import { trigger, transition, style, animate, query, animateChild, stagger } from '@angular/animations';
+import { stringify } from 'querystring';
```

Comment:
Is this import being used?

Reply:
Nope. This was a vscode auto complete auto import